README.md

```diff
@@ -1,29 +1,16 @@
-# Adobe I/O CodeLabs template
+# Getting Started with Adobe I/O Custom Event
 
-## Getting started 
+The I/O Events SDK is an open-source SDK for the use of third party customers. Third-party APIs in CSM requires bare minimum information from the users so that they can register providers and event metadata with ease, and register a webhook or journalling endpoint and listen to events. They can then start publishing messages to the event receiver for their org and then listen to those events by either polling via journalling or by allowing the webhook to be notified about events. 
```

Comment:
is it about I/O events itself or events SDK? in any case, you may want to add a link to the mentioned subject.

Reply:
fixed